tests/lowering/tensor_manipulation/test_unsqueeze.py

```diff
@@ -0,0 +1,131 @@
+import torch
+import torch_ttnn
+import unittest
+import ttnn
+import tt_lib
+
+# RuntimeError: TT_THROW @ ../ttnn/cpp/ttnn/operations/core.hpp:81: tt::exception
+# info: Unable to reshape a tensor in TILE_LAYOUT to non-tile height and width!
+# Please convert the input tensor to ROW_MAJOR_LAYOUT first.
+
+
+class Unsqueeze1Module(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    def forward(self, x, y):
+        return torch.unsqueeze(x, y)
+
+    def input_shapes(self):
+        return [(5, 2, 4, 3), 1]
+
+
+class Unsqueeze2Module(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    def forward(self, x, y):
+        return torch.unsqueeze(x, y)
+
+    def input_shapes(self):
+        return [(5, 2, 4), 1]
+
+
+class Unsqueeze3Module(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    def forward(self, x, y):
+        return torch.unsqueeze(x, y)
+
+    def input_shapes(self):
+        return [(5, 2, 4, 3), -2]
```

Comment:
I think this should be a single module. Input shapes should not be inside. This will help to eliminate duplication

Reply:
I tried with a single module creating class objects from one class module, but for some reason it errors out. @kevinwuTT also faced similar issues earlier. Multiple modules exist for previous tests, we can keep this for now.